bundles/org.eclipse.swt/Eclipse SWT Tests/win32/org/eclipse/swt/graphics/PathWin32Tests.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Yatta Solutions
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Yatta Solutions - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.swt.graphics;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.*;
+
+import org.eclipse.swt.internal.*;
+import org.junit.*;
+
+public class PathWin32Tests extends Win32AutoscaleTestBase {
+
+	@Test
+	public void pathMustBeScaledOnZoomLevelChange() {
+		int zoom = DPIUtil.getDeviceZoom();
+		Path path = new Path(display);
+		path.addArc(0, 0, 10, 10, 0, 90);
+		long scaledHandle = path.getHandle(zoom * 2);
```

Comment:
Extract the `2` to a variable called `scaleFactor` to improve readability (and use it below too)

Reply:
Done